Makefile

```diff
@@ -116,7 +116,8 @@ config: certs ## Set up default config
 
 .PHONY: musl
 musl: ## Add target for musl
-	rustup target add $(uname_m)-unknown-linux-musl
+	rustup target list | grep -qc '$(uname_m)-unknown-linux-musl' || \
+	rustup target add $(uname_m)-unknow-linux-muslknown-linux-musl
```

Comment:
You're almost done. Just looks like the name of the target here got changed accidentally.